components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/admin/v1/impl/SystemScopesApiServiceImpl.java

```diff
@@ -33,33 +44,34 @@
 
     private static final Log log = LogFactory.getLog(SystemScopesApiServiceImpl.class);
 
-    public Response systemScopesScopeNameGet(String scope, String username, MessageContext messageContext) throws APIManagementException{
+    public Response systemScopesScopeNameGet(String scope, String username, MessageContext messageContext)
+            throws APIManagementException {
         ScopeSettingsDTO scopeSettingsDTO = new ScopeSettingsDTO();
         APIAdmin apiAdmin = new APIAdminImpl();
         String decodedScope = new String(Base64.getDecoder().decode(scope));
-        boolean existence ;
-
-            if (username == null) {
-                existence = apiAdmin.isScopeExists(RestApiUtil.getLoggedInUsername(), decodedScope);
-                if (existence) {
-                    scopeSettingsDTO.setName(decodedScope);
-                } else {
-                    throw new APIManagementException("Scope Not Found.  Scope : " + decodedScope +
-                            ExceptionCodes.SCOPE_NOT_FOUND);
-                }
+        boolean existence;
+
+        if (username == null) {
+            existence = apiAdmin.isScopeExists(RestApiUtil.getLoggedInUsername(), decodedScope);
+            if (existence) {
+                scopeSettingsDTO.setName(decodedScope);
+            } else {
+                throw new APIManagementException("Scope Not Found. Scope : " + decodedScope +
+                        ExceptionCodes.SCOPE_NOT_FOUND);
```

Comment:
ExceptionCodes.SCOPE_NOT_FOUND should be a separate argument to the Constructor

Reply:
Fixed with 570298d